module/spl/spl-kstat.c

```diff
@@ -530,6 +530,18 @@ __kstat_set_raw_ops(kstat_t *ksp,
 }
 EXPORT_SYMBOL(__kstat_set_raw_ops);
```

Comment:
I didn't convert anything that iterates over an array or hash table, because the existing interface provided for raw kstats works pretty nicely for them, so those callers still remain.